privaterelay/templates/home.html

```diff
@@ -36,7 +36,10 @@ <h2 class="how-it-works-header text-center">{% ftlmsg 'how-it-works-headline' %}
             <div class="step-text step-1">
               <h3 class="step-headline ff-Met">{% ftlmsg 'how-it-works-step-1-headline' %}</h3>
               <p class="step-body">
-                {% ftlmsg 'how-it-works-step-1-copy-html' url='https://addons.mozilla.org/firefox/addon/private-relay/' attrs='target="_blank" rel="noopener" class="text-link"' %}
+                <a href="https://addons.mozilla.org/firefox/addon/private-relay/" target="_blank" rel="noopener" class="text-link">
+                  {% ftlmsg 'how-it-works-step-1-link' %}
+                </a>
```

Comment:
This underlines the space after the `.`.
<img width="331" alt="image" src="https://user-images.githubusercontent.com/71928/127537061-c5159c41-ae37-4d43-a807-52e4db362c5b.png">


```suggestion
                <a href="https://addons.mozilla.org/firefox/addon/private-relay/" target="_blank" rel="noopener" class="text-link">{% ftlmsg 'how-it-works-step-1-link' %}</a>
```

Reply:
Revised! 

<img width="325" alt="image" src="https://user-images.githubusercontent.com/2692333/127539104-00564ecc-6e92-4d5d-82ff-3d0c135d0836.png">